javascriptv3/example_code/ses/README.md

```diff
@@ -1,56 +1,74 @@
-
-4. Run sample code:
-```
-cd src
-node [example name].js // For example, node createreceiptfilter.js
-```
-## Resources
-- [AWS SDK for JavaScript v3](https://github.com/aws/aws-sdk-js-v3) 
-- [AWS SDK for JavaScript v3 Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/ses-examples.html) 
-- [AWS SDK for JavaScript v3 API Reference Guide](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-ses/index.html) 
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+- [Create a receipt filter](./src/ses_createreceiptfilter.js) (`CreateReceiptFilter`)
+- [Create a receipt rule](./src/ses_createreceiptrule.js) (`CreateReceiptRule`)
+- [Create a receipt rule set](./src/ses_createreceiptruleset.js) (`CreateReceiptRuleSet`)
+- [Create a template](./src/ses_createtemplate.js) (`CreateTemplate`)
+- [Delete an identity](./src/ses_deleteidentity.js) (`DeleteIdentity`)
+- [Delete a receipt filter](./src/ses_deletereceiptfilter.js) (`DeleteReceiptFilter`)
+- [Delete a receipt rule](./src/ses_deletereceiptrule.js) (`DeleteReceiptRule`)
+- [Delete a receipt rule set](./src/ses_deletereceiptruleset.js) (`DeleteReceiptRuleSet`)
+- [Delete a template](./src/ses_deletetemplate.js) (`DeleteTemplate`)
+- [Get a template](./src/ses_gettemplate.js) (`GetTemplate`)
+- [List all identities](./src/ses_listidentities.js) (`ListIdentities`)
+- [List all receipt filters](./src/ses_listreceiptfilters.js) (`ListReceiptFilters`)
+- [List all templates](./src/ses_listtemplates.js) (`ListTemplates`)
+- [Send a bulk templated email](./src/ses_sendbulktemplatedemail.js) (`SendBulkTemplatedEmail`)
+- [Send an email](./src/ses_sendemail.js) (`SendEmail`)
+- [Send a templated email](./src/ses_sendtemplatedemail.js) (`SendTemplatedEmail`)
+- [Update a template](./src/ses_updatetemplate.js) (`UpdateTemplate`)
+- [Verify a domain identity](./src/ses_verifydomainidentity.js) (`VerifyDomainIdentity`)
+- [Verify an email identity](./src/ses_verifyemailidentity.js) (`VerifyEmailIdentity`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- Coming soon.
+
+### Cross-service examples
+
+Sample applications that work across multiple AWS services.
+
+- Coming soon.
+
+## Run the examples
+
+1. `npm i`
+1. `node <path/to/example.js>`
+
+### Prerequisites
+
+- [Set up AWS SDK for JavaScript](../README.rst)
+
+## Tests
+
+⚠️ Running the tests might result in charges to your AWS account.
+
+1. `cd ../tests/ses`
+1. `npm test`
+
+## Additional resources
+
+- [SES Developer Guide](https://docs.aws.amazon.com/ses/latest/dg/Welcome.html)
+- [SES API Reference](https://docs.aws.amazon.com/ses/latest/APIReference/Welcome.html)
+- [SES Client - AWS SDK for JavaScript v3](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-ses/index.html)
```

Comment:
SES Client - AWS SDK for JavaScript v3 --> Amazon SES Client - AWS SDK for JavaScript (v3)